Acquisitions in swaps (#5082)
rotki/rotki: docs/changelog.rst

* :bug:`5051` Windows users with a premium subscription should be able to upload their user DB for backup to our server properly again.
* :bug:`4821` Provide a fix for an infinite loop querying FTX trades if many trades were made in a short time.
* :bug:`5073` Rotki won't fail to start at an edge case of starting the app again after more than a year when a particular global DB table that is expected is missing.

> * :bug:`-` Acquisitions in swaps should no longer be taxable in PnL reports.

Reviewer: ```suggestion
* :bug:`-` Acquisitions in swaps won't be taxable in PnL reports.
```
